codacy/values.yaml

```diff
@@ -263,14 +263,10 @@ codacy-ingress:
   create: true
   ingress:
     hostname:
-      app: <codacy-app.dns.internal>
-      api: <codacy-api.dns.internal>
+      app: "codacy.example.com"
```

Comment:
if this line and the one below change, then you must update the .doks/Makefile `update_ingress_values` `sed` commands to match the changed syntax

Reply:
we should never ever edit this document. I will remove the line that edits this document